docs/TROUBLESHOOTING.md

```diff
@@ -177,32 +177,39 @@ to
 ```
 WebDialog.RequestsDialogBuilder requestsDialog = (new WebDialog.RequestsDialogBuilder(me.cordova.getActivity(), Session.getActiveSession(), paramBundle)).setOnCompleteListener(dialogCallback);
 if (cordova.getActivity().getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
-	requestsDialog.setTheme(android.R.style.Theme_Wallpaper_NoTitleBar_Fullscreen);
+  requestsDialog.setTheme(android.R.style.Theme_Wallpaper_NoTitleBar_Fullscreen);
 }
 requestsDialog.build().show();
 ```
 
 ## iOS
 ### Missing FacebookConnectPlugin
 - Problem:
-	- `CDVPlugin class FacebookConnectPlugin (pluginName: facebookconnectplugin) does not exist.`
+  - `CDVPlugin class FacebookConnectPlugin (pluginName: facebookconnectplugin) does not exist.`
 - Solution:
-	1. Open up Xcode
-	2. Go to "Build Phases"
-	3. Ensure that the following file is added under "Compile Sources":
-		- `FacebookConnectPlugin.m`
-	4. Ensure that the following is added under "Link Binary With Libraris":
-		- `FacebookSDK.framework`
-		- `libsqlite3.dylib`
-		- `Social.framework`
-		- `Accounts.framework`
-		- `Security.framework`
+  1. Open up Xcode
+  2. Go to "Build Phases"
+  3. Ensure that the following file is added under "Compile Sources":
+    - `FacebookConnectPlugin.m`
+  4. Ensure that the following is added under "Link Binary With Libraris":
+    - `FacebookSDK.framework`
+    - `libsqlite3.dylib`
+    - `Social.framework`
+    - `Accounts.framework`
+    - `Security.framework`
 
 Cordova and plugman seems to have some problems adding frameworks etc. when re-installing/upgrading plugins.
 
 
 ### Login Always Opens Web Dialog Even Though Native App is Installed
 - Problem:
-	- Calling the login function always opens the web dialog even though the native app is installed on the device.
+  - Calling the login function always opens the web dialog even though the native app is installed on the device.
 - Solution:
-	- Switch "deep linking" to ON in the Facebook developer website settings for your application
+  - Switch "deep linking" to ON in the Facebook developer website settings for your application
+
+### Login dialog doen't show on iOS10
+- Problem:
+  - Calling the login function doen't do anything, No errors are thrown and no dialogs are shown.
+- Solution:
+  - iOS 10 requires additional entries in the Content-Security-Policy meta tag, namely gap://ready and file:. After adding this, Content-Security-Policy should looks like this: ```sh<meta http-equiv="Content-Security-Policy" content="default-src * gap://ready; style-src 'self' 'unsafe-inline'; img-src 'self' data:; script-src * 'unsafe-inline' 'unsafe-eval'">```
```

Comment:
It's updated automatically, as long as you make more commits to the same branch you created. With that, no, you don't need to recreate the PR.